### PR TITLE
zh_trans: kubeadm_alpha.md

### PR DESCRIPTION
**Pull request by zgfh**
zh_trans: /docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+
+实验子命令尚未完成全部功能。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha.md]: 功能尚不完整的实验性子命令

Comment (zgfh) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha.md]: 感谢指正

**Comment by Hartmarken348** [comment]
Adm alpha 11902

**Pull request merged**